vue/components/ui/generic-field-platforme/generic-field-platforme.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="generic-field-container">
+        <img class="icon" v-bind:src="iconSrc" v-if="iconSrc" v-show="iconVisible" />
+        <input
+            class="generic-field-platforme"
+            v-bind:class="[iconVisible ? '' : 'icon-invisible', disabled ? 'disabled' : '']"
+            v-bind:type="type"
+            v-bind:value="value"
+            v-bind:disabled="disabled"
+            v-bind:placeholder="placeholder"
+            v-bind:required="required"
+            v-bind:autofocus="autofocus"
+            v-bind:style="'width: ' + width + 'px; ' + 'height: ' + height + 'px;'"
+            ref="input"
+            v-on:input="$emit('update:value', $event.target.value)"
+            v-on:focus="focused = true"
+            v-on:blur="focused = false"
+            v-on:keydown.esc="blur()"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.generic-field-container {
+    align-items: center;
+    display: inline-flex;
+}
+
+.disabled:hover {
+    background-color: #dadada;
+    border-color: #ff9b9b !important;

```

Comment:
Remove the `!important`, that's only to be used in extreme cases and should be avoided